src/client/app/containers/LineChartContainer.ts

```diff
@@ -21,6 +22,7 @@ function mapStateToProps(state: State) {
 			const readingsData = byMeterID[timeInterval.toString()];
 			if (readingsData !== undefined && !readingsData.isFetching) {
 				const label = state.meters.byMeterID[meterID].name;
+				const colorID = state.meters.byMeterID[meterID].id;
```

Comment:
I checked an meterID and colorID are the same. Thus, calling state.meters.byMeterID[meterID].id returns the value meterID. Thus, this can be changed to
const colorID = meterID;
The same applies to groupID below and both of these changes should be applied to BarChartContainer.ts.

Reply:
Thanks for that recommendation. I've made the quick requested changes.